Jerkinator/SecondRollFrontend
# 58 link game ad page and wishlist to rtd modal

Created a wishlist icon that adds a game to the users wishlist when clicked, placed in GameAdDetails.
Added a button on proflie page, "Visa önskelista". 

To test, checkout branch: 
58-link-game-ad-page-and-wishlist-to-rtd-modal

Make sure you are logged in, then go to a game ad and click the heart icon. In the console it should say "added" and the game id will now be on users wishlist. 

While logged in and on your profile page, click the button "Visa önskelista" to be redirected to your wishlist.
